index.js

```diff
@@ -25,7 +25,13 @@ const getJestConfig = function(cliConfigParameter) {
   return {};
 };
 
+let isSilent = false;
+
 const log = function(message, status = "log") {
+  if (isSilent) {
```

Comment:
`log` function can be check `silent` property of `options` instead of `isSilent` variable.

```
if (ioptions.sSilent) return;
```